tools/installers/install_mfa.sh

```diff
@@ -2,22 +2,27 @@
 
 set -euo pipefail
 
-if [ $# != 1 ]; then
-    echo "Usage: $0 true/false"
-    exit 1;
-fi
+# This is the last stable version. MFA 3.0 depends on an unstable kaldi version that creates errors
```

Comment:
I would like to confirm. Did you test the installation and execution with conda? Because, I remember it was generating an issue w.r.t the PostgreSQL or another kind of application. That is the reason for using pip installation.
But, if it is already fixed, the ok